syntax = "proto3";

package proto.lidar;

message Scan2d {
    float position_x = 1;
    float position_y = 2;
}

message Scan3d {
    float position_x = 1;
    float position_y = 2;
    float position_z = 3;
}

message PointCloud2d {
    string lidar_id = 1;
    repeated Scan2d ranges = 2;
}

message PointCloud3d {
    string lidar_id = 1;
    repeated Scan3d ranges = 2;
}

message Health {
    int32 status = 1;
    string message = 2;
}

